apps/prairielearn/src/models/course.sql

```diff
@@ -110,3 +110,52 @@ SELECT
   *
 FROM
   updated_row;
+
+-- BLOCK insert_course
+WITH
+  new_row AS (
+    INSERT INTO
+      pl_courses AS c (
+        short_name,
+        title,
+        display_timezone,
+        path,
+        repository,
+        branch,
+        institution_id
+      )
+    VALUES
+      (
+        $short_name,
+        $title,
+        $display_timezone,
+        $path,
+        $repository,
+        $branch,
+        $institution_id
+      )
+    RETURNING
+      *
+  ),
+  audit_log AS (
```

Comment:
Typically audit logs would be handled by calling `insertAuditLog` in TS from within a transaction that does the actual insertion/deletion/etc.

Regardless of if you do that, it'd be nice to also populate `institution_id` and `course_id` in the `audit_logs` table.

Reply:
> Typically audit logs would be handled by calling `insertAuditLog` in TS from within a transaction that does the actual insertion/deletion/etc.

I was unaware that this function existed, I just checked and it seems it's only used in code I haven't touched before (EE code and institution administrator stuff). I can add it.

Reply:
I have replaced it with the models function in aa30bcfd6. I also updated the audit log entry in the delete function in c89f4f70f.